.github/workflows/on-pullrequest.yml

```diff
@@ -58,7 +68,7 @@ jobs:
         uses: actions/upload-artifact@0b7f8abb1508181956e8e162db84b466c27e18ce # v3.1.2
         with:
           name: csharp-tests
-          path: TestResults/unit-tests.trx
+          path: "**/*.trx"
```

Comment:
This modification is already working, scroll to [Artifacts on this page](https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-AdminAPI/actions/runs/6949672373) to see the uploaded test results.